perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/MembersManagerMethod.java

```diff
@@ -1541,6 +1545,7 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 								ac.getGroupById(parms.readInt("group")),
 								parms.readList("attrsNames", String.class),
 								parms.readList("allowedStatuses", String.class),
+								parms.readList("allowedGroupStatuses", String.class),
```

Comment:
I would suggest making the `allowedGroupStatuses` parameter optional. This method might be already used in multiple GUI applications and this would require all of them to be changed. First, check if the `allowedGroupStatuses` is present in the params and if so, read it. If not, use some empty list.

Please make this change for the other method as well.

Reply:
The parameter `allowedGroupStatuses` is now optional.